tests/app/service/send_notification/test_send_notification.py

```diff
@@ -96,7 +87,11 @@ def test_should_reject_bad_phone_numbers(notify_api, sample_template, mocker):
 
 
 @pytest.mark.parametrize(
-    "template_type, to", [(SMS_TYPE, "+447700900855"), (EMAIL_TYPE, "ok@ok.com")]
+    "template_type, to",
+    [
+        (TemplateType.SMS, "+447700900855"),
+        (TemplateType.EMAIL, "ok@ok.com"),
+    ],
```

Comment:
Given our recent experience with test phone numbers, we might want to change these while we're at it to known and valid test values.  I realize this is just refactoring what was already here, but two birds and all... :-)